ifsbench/data/namelisthandler.py

```diff
@@ -164,7 +166,86 @@ def execute(self, wdir, **kwargs):
         debug(f"Modify namelist {input_path}.")
         namelist = f90nml.read(input_path)
 
+        namelist.uppercase = True
+        namelist.end_comma = True
+
         for override in self.overrides:
             override.apply(namelist)
 
         namelist.write(output_path, force=True)
+
+class SanitiseMode(Enum):
+    """
+    Specify, how duplicated entries in a namelist should be sanitised.
+    """
+
+    #: For multiply defined namelist groups, retain only the first.
+    FIRST = 'first'
+
+    #: For multiply defined namelist groups, retain only the last.
+    LAST = 'last'
+
+    #: For multiply defined namelist groups, merge variable definitions from
+    #: all groups. Conflicts are resolved by using the first occurence of a
+    #: variable.
+    MERGE_FIRST = 'merge_first'
+
+    #: For multiply defined namelist groups, merge variable definitions from
+    #: all groups. Conflicts are resolved by using the last occurence of a
+    #: variable.
+    MERGE_LAST = 'merge_last'
+
+class NamelistSanitiseHandler(DataHandler):
```

Comment:
How does this compare to IFSNameList which also has sanitising? Can that be deleted?